src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo:React.FC<Props> = ({ user }) => (
+  <a
```

Comment:
You are not checking if user exists before rendering the component, which means this component will render an empty anchor tag if no user is supplied. It would be better to conditionally render this component based on whether there is a user or not.